care/facility/api/serializers/facility.py

```diff
@@ -83,6 +97,8 @@ class FacilitySerializer(FacilityBasicInfoSerializer):
     read_cover_image_url = serializers.URLField(read_only=True)
     # location = PointField(required=False)
     features = serializers.MultipleChoiceField(choices=FEATURE_CHOICES)
+    patient_count = serializers.SerializerMethodField()
```

Comment:
You don't need to redefine this since it's already present in the parent class.

Reply:
Noted. Changes made.